src/main.cpp

```diff
@@ -1,29 +1,32 @@
 #include <Arduino.h>
-#include "controller.hpp"
+
```

Comment:
include config.hpp

Reply:
If anything from `config.hpp` is needed in main, it should be included. Often the compiler figures out the missing dependencies, but it's good practice to include all necessary headers explicitly.

Reply:
Ok, `main.cpp` doesn't directly use anything from `config.hpp`, that's why I didn't include it.
I'm going to move the `main.cpp` into the `hardware/tests/hw_test_motors.cpp` now